lnrpc/rpc.proto

```diff
@@ -1408,7 +1466,8 @@ message NodeInfo {
     LightningNode node = 1 [json_name = "node"];
 
     uint32 num_channels = 2 [json_name = "num_channels"];
-    int64 total_capacity = 3 [json_name = "total_capacity"];
+
+    repeated NodeChainInfo chains =4 [json_name = "chains"];
```

Comment:
Add space after `=` and change the index from 4 to 3.

Reply:
Index is 4 on purpose. Not to create issues for RPC users which may use field index =3

Reply:
Ah OK, so for backwards compatibility essentially?